docs/about/compatibility/openmp.md

```diff
@@ -297,7 +297,7 @@ The file `veccopy-ompt-target-tracing.c` simulates how a tool initiates device
 activity tracing. The file `callbacks.h` shows the callbacks registered and
 implemented by the tool.
 
-### Floating point atomic operations
+### Floating-point atomic operations
 
 The MI200-series GPUs support the generation of hardware floating-point atomics
 using the OpenMP atomic pragma. The support includes single- and
```

Comment:
```suggestion
using the OpenMP atomic pragma. The support includes single and
```

Reply:
The hyphen should be there (single-precision).